controllers/managedocs_controller.go

```diff
@@ -1042,6 +1044,14 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 			}
 		}
 
+		// getting clusterID
+		clusterVersion := &configv1.ClusterVersion{}
+		if err := r.UnrestrictedClient.Get(context.Background(), types.NamespacedName{Name: "version"}, clusterVersion); err != nil {
+			return err
+		}
```

Comment:
Why do we need the `UnrestrictedClient`? and even if we need it we have a `r.UnrestrictedClient` that you can use

Reply:
@nb-ohad I think you meant r.unrestrictedGet, right?

Reply:
Correct, updating the comment